lib/packages/http/README.md

```diff
@@ -0,0 +1,127 @@
+# @burstjs/http
+
+Generic HTTP client that is used as a network provider for @burst/core.
+
+## Installation
+
+Install using [npm](https://www.npmjs.org/):
+
+```
+npm install @burstjs/http
+```
+
+or using [yarn](https://yarnpkg.com/):
+
+``` yarn
+yarn add @burstjs/http
+```
+
+## API Reference
+## Classes
+
+<dl>
+<dt><a href="#HttpError">HttpError</a></dt>
+<dd><p>HttpError class</p>
+<p>Thrown on HTTP errors</p></dd>
+<dt><a href="#HttpImpl">HttpImpl</a></dt>
+<dd></dd>
+<dt><a href="#HttpMock">HttpMock</a></dt>
+<dd><p>Http Mocker for easy to http testing using Jest</p>
+<p>When using this mocking helper you need to call <code>Http.onGet()</code>
+before Http instance is created</p></dd>
+</dl>
+
+<a name="HttpError"></a>
+
+## HttpError
+<p>HttpError class</p>
+<p>Thrown on HTTP errors</p>
+
+**Kind**: global class  
+<a name="HttpImpl"></a>
+
+## HttpImpl
+**Kind**: global class  
+<a name="new_HttpImpl_new"></a>
+
+### new HttpImpl(baseURL)
+<p>Creates your Http client</p>
+
+
+| Param | Description |
+| --- | --- |
+| baseURL | <p>The baseUrl, i.e host url</p> |
+
+<a name="HttpMock"></a>
+
+## HttpMock
+<p>Http Mocker for easy to http testing using Jest</p>
```

Comment:
Huh? Where's the mock builder?

Reply:
It might have got confused. It doesn't seem to like code in comment blocks. There are also two httpMock classes, so that could be throwing it off too.